pkg/destroy/gcp/filestore.go

```diff
@@ -0,0 +1,82 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"google.golang.org/api/file/v1"
+)
+
+func (o *ClusterUninstaller) filestoreParentPath() string {
+	return fmt.Sprintf("projects/%s/locations/-", o.ProjectID)
+}
+
+func (o *ClusterUninstaller) clusterFilestoreLabelFilter() string {
+	return fmt.Sprintf("labels.kubernetes-io-cluster-%s = \"owned\"", o.ClusterID)
```

Comment:
nit: Recently a new constant was introduced in `github.com/openshift/installer/pkg/constants/gcp` package for OCP default GCP label format `ClusterIDLabelFmt = "kubernetes-io-cluster-%s"`. I feel we could make use of it.